internal/datanode/services.go

```diff
@@ -214,6 +215,10 @@ func (node *DataNode) CompactionV2(ctx context.Context, req *datapb.CompactionPl
 		return merr.Success(), nil
 	}
 
+	if req.GetBeginLogID() == 0 {
+		return merr.Status(merr.WrapErrParameterInvalidMsg("invalid beginLogID")), nil
```

Comment:
log what is the beginLodID is?

Reply:
and why do we need to add some check here? Seems to be unrelated to what we want to solve here.

And how do we become compatible with existing?

Reply:
@bigsheeper  @czs007 
are we use the this GetBeginLogID somewhere? 
the name is here is a little bit hacky